datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -44,11 +44,14 @@ def clean(ctx, check, compiled_only, all_matches, force, verbose):
     If `check` is not specified, the current working directory will be used.
 
     All `*.pyc`/`*.pyd`/`*.pyo`/`*.whl` files and `__pycache__` directories will be
-    removed. Additionally, the following patterns will be removed from the root of
+    removed.
+
+    Additionally, the following patterns will be removed from the root of
     the path: `.cache`, `.coverage`, `.eggs`, `.pytest_cache`, `.tox`, `build`,
     `dist`, and `*.egg-info`.
     """
-    force_clean_root = False
+    force_clean_root = None
```

Comment:
why None?

Reply:
Sort of a hack indeed, but now stale on the latest version of this PR.